packages/create-vite/template-ember-ts/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "vite-ember-ts-starter",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Small description for app-template goes here",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "type": "module",
+  "imports": {
+    "#src/*": "./src/*",
+    "#config": "./src/config.ts",
+    "#components/*": "./src/components/*"
+  },
+  "exports": {
```

Comment:
Is this needed?

Reply:
I'm not sure -- lemme ask my team why it was added

Reply:
it is needed so that folks can import using the module name